test/integration_test/pdb_test.go

```diff
@@ -101,11 +101,13 @@ func OverridePDBUsingValidAnnotation(tc *types.TestCase) func(*testing.T) {
 
 		k8snodecount, err := ci_utils.GetK8sNodeCount()
 		require.NoError(t, err)
-		// Assuming 1 node is dedicated control plane node
-		k8snodecount = k8snodecount - 1
+		// Assuming 3 nodes are dedicated control plane nodes
```

Comment:
We shouldn't assume there are 3 control plane nodes, why not just get the count of master nodes?

Reply:
Is there a way to get the actual count?

Reply:
@nikolaypopov made the requested changes and tested it, here is the link - https://jenkins.pwx.dev.purestorage.com/job/Operator/job/op-pdb-ocp/11/console

Reply:
Yes, there is, you can loop through nodes and find which are the master nodes by using this `sched-ops` method https://github.com/portworx/sched-ops/blob/master/k8s/core/nodes.go#L151

Reply:
Yes I have used this same function. The run passed with these changes